CHANGELOG.md

```diff
@@ -8,6 +8,9 @@
 - Debug from .csproj and .sln [#5876](https://github.com/dotnet/vscode-csharp/issues/5876)
 
 ## Latest
+* Update Roslyn version to 4.8.0-3.23470.7 (PR: [#6408](https://github.com/dotnet/vscode-csharp/pull/6408))
+  * Update NuGet version to fix issues loading projects with .NET 8 RC2 (PR: [#70023](https://github.com/dotnet/roslyn/pull/70023))
+  * Update MSBuildLocator version to fix issues loading projects when only .NET6 is installed (PR: [#70038](https://github.com/dotnet/roslyn/pull/70038))
 * Update Roslyn version (PR: [#6389](https://github.com/dotnet/vscode-csharp/pull/6389))
```

Comment:
Delete this line for Roslyn updates, maybe? Last time we did this I just merged the sections and didn't list the Roslyn update PR. I mean it's a PR that changes a number...not exciting.